core/presentation/ui/src/main/java/com/hyunjung/core/presentation/ui/UiText.kt

```diff
@@ -0,0 +1,29 @@
+package com.hyunjung.core.presentation.ui
+
+import android.content.Context
+import androidx.annotation.StringRes
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+
+sealed interface UiText {
+    data class DynamicString(val value: String) : UiText
+    class StringResource(
+        @StringRes val id: Int,
+        val args: Array<Any> = arrayOf()
+    ) : UiText
+
+    @Composable
```

Comment:
```suggestion
    @Composable @ReadOnlyComposable
```

요거 추천합니당! 

https://developer.android.com/reference/kotlin/androidx/compose/runtime/ReadOnlyComposable